lib/GADS/Schema/Result/File.pm

```diff
@@ -183,6 +189,26 @@ sub export_hash
 # Created by DBIx::Class::Schema::Loader v0.07043 @ 2015-11-13 16:02:57
 # DO NOT MODIFY THIS OR ANYTHING ABOVE! md5sum:266q8eJmPiCjcNhwddaUkw
 
+sub _build_recordsource { 'File'; }
+sub _build_valuefield { undef; }
+
+# I've done this to override the role - I don't know how else to best implement this!
```

Comment:
I know we discussed this, but I can't remember the reason for putting "purged" in a file but blanking all other fields?

Reply:
In order to show on cronology that a file was uploaded and when (sometimes the file is uploaded later than the record being created)